command/v7/unset_label_command_test.go

```diff
@@ -28,6 +34,49 @@ var _ = Describe("unset-label command", func() {
 		executeErr      error
 	)
 
+	checkBuildpackArgFunc := func() {
+		BeforeEach(func() {
+			cmd.BuildpackStack = "cflinuxfs3"
+		})
+
+		It("displays an argument combination error", func() {
+			argumentCombinationError := translatableerror.ArgumentCombinationError{
+				Args: []string{strings.ToLower(cmd.RequiredArgs.ResourceType), "--stack, -s"},
+			}
+
+			Expect(executeErr).To(MatchError(argumentCombinationError))
+		})
+
+	}
+	When("--help flag is set", func() {
```

Comment:
These tests should be moved to the integration tests. This is causing our units to go red (Travis is Red).

Reply:
It actually looks like the tests are present in the integrations and these here are redundant. It should be confirmed.

Reply:
Confirmed. I don't know how those tests got into the command layer unit tests but they don't belong there.